homeassistant/components/darksky/weather.py

```diff
@@ -67,10 +70,13 @@
         vol.Optional(CONF_MODE, default="hourly"): vol.In(FORECAST_MODE),
         vol.Optional(CONF_UNITS): vol.In(["auto", "si", "us", "ca", "uk", "uk2"]),
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(
+            CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL
+        ): cv.positive_int,
     }
 )
 
-MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=3)
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=CONF_SCAN_INTERVAL)
```

Comment:
This doesn't look correct. `CONF_SCAN_INTERVAL` is a string constant.

Reply:
It isn't correct, I'm looking into it now.